databases/week1/HomeworkExercises.sql

```diff
@@ -0,0 +1,88 @@
+--1 Find how many tasks are in the table
+SELECT COUNT(id)
+id
+FROM
+task;
+
+--2 Find out how many tasks do not have a valid due date
+SELECT COUNT(id)
+id
+FROM
+task
+WHERE 
+due_date IS NULL;
+
+--3 Find all the tasks that are marked as done
+SELECT
+*
+FROM
+task
+JOIN
+status
+ON task.status_id = status.id
+WHERE name LIKE "Done";
```

Comment:
WHERE status.name = 'DONE';

Pointers
1) Use prefix for columns for readability1 (status.name)
2) Use = operations and keep LIKE For pattern searches